internal/librarian/generate.go

```diff
@@ -235,12 +239,12 @@ func detectIfLibraryConfigured() (bool, error) {
 		}
 	}
 	// If the library doesn't exist, we don't use the repo at all.
-	libraryID := findLibraryIDByApiPath(pipelineState, flagAPIPath)
+	libraryID := findLibraryIDByApiPath(pipelineState, apiPath)
 	if libraryID == "" {
-		slog.Info(fmt.Sprintf("API path %s not configured in repo", flagAPIPath))
+		slog.Info(fmt.Sprintf("API path %s not configured in repo", apiPath))
 		return false, nil
 	}
 
-	slog.Info(fmt.Sprintf("API path %s configured in repo library %s", flagAPIPath, libraryID))
+	slog.Info(fmt.Sprintf("API path %s configured in repo library %s", apiPath, libraryID))
```

Comment:
If you're using slog, might as well go all-in on structured logging:
```
slog.Info("API configured", "path", apiPath, "library", libraryID)
```

Reply:
done